plugins/Scripts/Plugins/Registration/Correct_3D_drift.py

```diff
@@ -22,16 +22,42 @@
 from ij.gui import GenericDialog, YesNoCancelDialog, Roi
 from mpicbg.imglib.image import ImagePlusAdapter
 from mpicbg.imglib.algorithm.fft import PhaseCorrelation
-from org.scijava.vecmath import Point3i
-from org.scijava.vecmath import Point3f
+from javax.vecmath import Point3i
+from javax.vecmath import Point3f
```

Comment:
Is this intended? You're going backwards from `vecmath-1.6.0-scijava-2` to `vecmath-1.5.2`. For Fiji running with Java 8, you should keep using `org.scijava.vecmath`.